android-o/SafeBrowsing/Metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<SampleMetadata>
+	<ID>534E4EC4-76BA-46A8-A025-E26858CD59BD</ID>
+	<IsFullApplication>false</IsFullApplication>
+	<Level>Beginner</Level>
+	<Tags>Data, Media</Tags>
+	<SupportedPlatforms>Android</SupportedPlatforms>
+	<LicenseRequirement>Starter</LicenseRequirement>
+	<Brief>Demonstrates how to enable safe browsing.</Brief>
+	<Gallery>false</Gallery>
```

Comment:
I think we can set this to true.

@mgmclemore is there any reason for us to disable new samples in the Gallery by default, or is it alright for new samples to set this element to true when they are added?